### ISSUE TITLE
Compressor error when running locally

### ISSUE THREAD

**Issue by odscjames** [IATI/IATI-Standard-Website#780]
web-1            | CommandError: Compressor is disabled. Set the COMPRESS_ENABLED setting or use --force to override.

Anyone else?

I hand edit the entrypoint.sh file and delete that line, then am careful not to check it in. But maybe we can find a better solution?

**Comment by Bjwebb** [comment]
Yeah, I run into this. I normally end up running `export COMPRESS_ENABLED=1` before my docker commands. We should fix this though. I've not had chance to figure out why this isn't a problem with the dev and live deploys.

**Comment by odscjames** [comment]
Tried to have a quick look and it's not just a simple fix, as there is issues about whether it runs in test or not. For now, just doc'd so at least people know how to run this locally.
